src/components/Jobs/components/JobsList/JobsList.css

```diff
@@ -6,9 +6,8 @@
   margin-bottom: 5px;
 }
 
-.loading {
-  height: 60px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-}
\ No newline at end of file
+.isEmptyMessage {
+  display: block;
```

Comment:
Change the element to just `div` if block behavior is needed which is this and next line. I think emptyMessage is enough as it is not a modifier class.

Reply:
I agree that class name should be changed but because of peculiarities of List component which wraps empty message with <p> it throws the following warning (screenshot attached).
![image](https://user-images.githubusercontent.com/44395295/48842494-1bfcc980-ed9d-11e8-9dca-b811026fa055.png)

Reply:
Fine then.